src/vuexTemplates/store/modules/users.js

```diff
@@ -11,10 +11,10 @@ const getters = {
     return state.selectedUser;
   },
 };
-
+let userId = 0
 const actions = {
```

Comment:
Will replace the functionality of generating userId by using the UUID in another PR.